packages/nouns-subgraph/package.json

```diff
@@ -22,10 +22,10 @@
     "create:localnode": "graph create --node http://localhost:8020/",
     "deploy": "graph deploy --debug --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/",
     "deploy:localnode": "graph deploy --debug --node http://localhost:8020/ --ipfs http://localhost:5001/",
-    "deploy:hardhat": "yarn clean && yarn prepare:hardhat && yarn codegen && yarn create:localnode nounsdao/nouns-subgraph && yarn deploy:localnode nounsdao/nouns-subgraph",
+    "deploy:hardhat": "yarn clean && yarn prepare:hardhat && yarn codegen && graph create --node http://localhost:8020 nounsdao/nouns-subgraph-hardhat && graph deploy --debug --ipfs http://localhost:5001 --node http://localhost:8020 nounsdao/nouns-subgraph-hardhat",
```

Comment:
That is historical reasons:
- I added`deploy:hardhat` script before Nouns DAO added the same script.
- I keeped my implementation because graph name is differ.